README.rst

```diff
@@ -11,6 +10,14 @@ Fauna Python
 
 Python driver for `Fauna <https://fauna.com>`_.
 
+**Note**::
+
+    This driver is in beta release and not recommended for production use. 
+    It operates with the Fauna database service via an API which is also in 
+    beta release, and is not recommended for production use. This driver is
+    not compatible with v4 or earlier versions of Fauna. If you would like
+    to participate in the private beta program please contact product@fauna.com.
+    
```

Comment:
```suggestion
.. warning:: 
  This driver is in beta release and not recommended for production use. 
  It operates with the Fauna database service via an API which is also in 
  beta release, and is not recommended for production use. This driver is
  not compatible with v4 or earlier versions of Fauna. If you would like
  to participate in the private beta program please contact product@fauna.com.

```

while the rendering isn't as nice IMO we should adhere to [reST](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-warning)